src/services/configService.js

```diff
@@ -16,6 +18,12 @@ export class ConfigService {
       .map(toDeviceGroupsModel);
   }
 
+  /** Returns the azure map key for the account */
+  static getAzureMapKey() {
+    return Observable.of('') // TODO: Implement the real service call
+      .delay(2000);
```

Comment:
You can implement the call. The service call is not going to change from before.

Reply:
I can add it in it's own PR.